src/schema.ts

```diff
@@ -13,7 +14,7 @@ function getSQLVersion(connection: string): SQLVersion {
     } else if (/^mysql:\/\//i.test(connection)) {
         return SQLVersion.MYSQL
     } else {
-        throw new Error(`SQL version unsupported in connection: ${connection}`)
+        return SQLVersion.UNKNOWN
```

Comment:
If we leave it as is we have unreachable code. It will still throw an error [here](https://github.com/SweetIQ/schemats/blob/master/src/schema.ts#L26). Other options are to get rid of that default catch, or to just use the if/else directly rather than converting to an enum.